recipes/sdl2/run_test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+${PREFIX}/bin/sdl2-config --version
```

Comment:
Would drop this file and add the command to the test section mentioned above. This should give you Windows testing for free. 😄